plugins/GSdx/GSState.cpp

```diff
@@ -5252,7 +5252,7 @@ bool GSC_UrbanReign(const GSFrameInfo& fi, int& skip)
 {
 	if(skip == 0)
 	{
-		if(fi.TME && fi.FBP==0x0000 && fi.TBP0==0x3980 && fi.FPSM==fi.TPSM && fi.TPSM == PSM_PSMCT32 && fi.TPSM ==0  && fi.FBMSK == 0x0)
+		if(fi.TME && fi.FBP==0x0000 && fi.TBP0==0x3980 && fi.FPSM==fi.TPSM && fi.TPSM == PSM_PSMCT32 && fi.FBMSK == 0x0)
```

Comment:
Coverity reports dumb conditions. In this case TPSM was tested against 2 values. 0 and PSMCT32 (strangely coverity saw them as not egual)